monitor/src/web-api/web.api.service.ts

```diff
@@ -0,0 +1,25 @@
+import { Injectable } from "@nestjs/common";
+import { PoolClient } from "pg";
+import { ethers } from "ethers";
+import { CommonConfigService } from "src/common/common.config";
+
+import { MonitorConfigService } from "src/monitor.config/monitor.config.service";
```

Comment:
`MonitorConfigService`, `MONITOR_CONFIG` and `IncomingWebhook` are not necessary to import.